requirements.txt

```diff
@@ -1,6 +1,197 @@
--e git://github.com/sa2c/Rynner.git # Rynner
-cellprofiler>=4.2.0
-cellprofiler_core>=4.2.0
-timeago>=1.0.15
-wxPython>=4.1.0
-parsl>=1.1.0
\ No newline at end of file
+absl-py==1.0.0
+anyio==3.5.0
+argon2-cffi==21.3.0
+argon2-cffi-bindings==21.2.0
+asn1crypto==0.24.0
+asttokens==2.0.5
+astunparse==1.6.3
+atomicwrites==1.4.0
+attrs==21.4.0
+Babel==2.9.1
+backcall==0.2.0
+backports.functools-lru-cache==1.5
+bcrypt==3.1.6
+bleach==5.0.0
+boto3==1.20.48
+botocore==1.23.48
+cachetools==5.0.0
+cellh5==1.3.0
+CellProfiler==4.2.1
+cellprofiler-core==4.2.1
+centrosome==1.2.0
+certifi==2018.11.29
+cffi==1.15.0
+chardet==3.0.4
+charset-normalizer==2.0.11
+click==8.0.3
+cloudpickle==2.0.0
+colorama==0.4.4
+configparser==3.7.1
+contextlib2==0.5.5
+coverage==6.3.2
+cryptography==3.3.2
+cycler==0.10.0
+dask==2022.4.0
+DateTime==4.3
+debugpy==1.5.1
+decorator==4.3.2
+defusedxml==0.7.1
+deprecation==2.0.6
+dill==0.3.4
+docutils==0.15.2
+entrypoints==0.4
+enum34==1.1.6
+executing==0.8.3
+flatbuffers==2.0
+fsspec==2022.3.0
+future==0.18.2
+gast==0.5.3
+globus-sdk==3.3.1
+google-auth==2.6.0
+google-auth-oauthlib==0.4.6
+google-pasta==0.2.0
+grpcio==1.43.0
+h5py==3.2.1
+hmmlearn==0.2.6
+idna==2.8
+imageio==2.14.1
+imglyb==2.0.0
+importlib-metadata==4.10.1
+importlib-resources==5.4.0
+inflect==2.1.0
+iniconfig==1.1.1
+ipaddress==1.0.22
+ipykernel==6.9.1
+ipython==8.1.1
+ipython-genutils==0.2.0
+javabridge==1.0.19
+jedi==0.18.1
+jgo==1.0.3
+Jinja2==3.0.3
+jmespath==0.9.3
+joblib==1.1.0
+JPype1==1.3.0
+json5==0.9.6
+jsonschema==4.4.0
+jupyter-client==7.1.2
+jupyter-core==4.9.2
+jupyter-server==1.13.5
+jupyterlab==3.3.0
+jupyterlab-pygments==0.1.2
+jupyterlab-server==2.10.3
+keras==2.8.0
+Keras-Preprocessing==1.1.2
+kiwisolver==1.0.1
+libclang==13.0.0
+libsubmit @ git+https://github.com/MYPei/libsubmit.git@dd93180873010710917ed8ad6bfc03b64b8836a1
```

Comment:
We should move this fork of `libsubmit` into the `sa2c` organisation rather than having a dependency on an individual who no longer works for our team.

Reply:
I've created a `libsubmit` repo in `sa2c` and I am working on simplifying the requirements list. The `windows-patch` branch in the `Rynner` repo was updated as well.